### PR TITLE
Upgraded the Organisations Card

### PR DESCRIPTION
**Pull request by swetagit186**
upgraded the organisations card with the relevant screenshots:-
Before hovering:
Screenshot 2023-10-12 at 5 18 12 PM
After hovering:-
Screenshot 2023-10-12 at 5 18 24 PM

### CODE REVIEW

**Review by EshaanAgg — CHANGES_REQUESTED** [review]
The change in the organization card is super awesome! Thanks.
Just need some minor changes.

File: src/pages/org/[orgID].astro [hunk 1]
```diff
@@ -2,6 +2,10 @@
 import Layout from "../../layouts/Layout.astro";
 import { ProjectNavbar } from "../../components/organization/ProjectNavbar";
 
+interface Organization {
+  id:number;
+}
+
```
[the remark this answers is not in the document]
Comment (swetagit186) [reply src/pages/org/[orgID].astro:8]: actually this  file was showing an error  so shravani did this on my branch by mistake and i overlooked it

File: src/pages/index.astro [hunk 2]
```diff
@@ -7,12 +7,42 @@ const response = await fetch(`${API_BASE_URL}/orgs`);
...
+      }
+    </ul>
+  </div>
 </Layout>
+
+<!-- grid grid-cols-4  ml-[120px] mr-[120px] gap-20  -->
+
+<!-- backgroundGradient={<div className="codeblock1 absolute"></div>} -->
+<style>
+  .codeblock {
```
[the remark this answers is not in the document]
Comment (swetagit186) [reply src/pages/index.astro]: Done

File: src/pages/org/[orgID].astro (hunk 1 above)
[the remark this answers is not in the document]
Comment (EshaanAgg) [reply src/pages/org/[orgID].astro:8]: Can you remove it?

[the remark this answers is not in the document]
Comment (swetagit186) [reply src/pages/org/[orgID].astro]: Done

**Pull request merged**